lib/scrabble/player.rb

```diff
@@ -0,0 +1,50 @@
+module Scrabble
+  class Player
+    attr_accessor :name, :played_words, :playing
+
+    def initialize(name)
+      @name = name
+      @played_words = []
+      @playing = true
+    end
+
+    def play(word)
+      if @playing
+        @played_words.push(word)
+      else
+        return false
+      end
+    end
+
+    def plays
+      return @played_words
```

Comment:
Since Ruby automatically returns whatever the last line of a method is, you don't need to explicitly say `return`.